drivers/mpu9250/MPU9250.cpp

```diff
@@ -248,9 +300,56 @@ int MPU9250::mpu9250_deinit()
 }
 
 int MPU9250::start()
+{
+#if defined(__DF_BBBLUE)
+	// on BBBLUE, MPU9250 and BMP280 are on the same I2C bus
```

Comment:
Sounds like a bad design choice. Do you get good flight performance on that board?